rwkv.cpp

```diff
@@ -189,7 +189,7 @@ enum rwkv_type {
 
 #define GGML_TYPE_UNKNOWN GGML_TYPE_COUNT
 
-static const enum ggml_type rwkv_type_to_ggml[TYPE_COUNT + 1] = {
+extern const enum ggml_type rwkv_type_to_ggml[TYPE_COUNT + 1] = {
```

Comment:
(just asking) What does `extern` do?

Reply:
It allows the variables to be used from FFI (which I am using to parse model files in rust by using rwkv.cpp as a library)

I am not sure if you want to allow this

Reply:
Hmm, maybe they should be in the header file then, as an official part of the API...

Reply:
Ok, we can add then to the header any time. Let's keep `extern`